src/Symfony/Component/Messenger/Asynchronous/Transport/WrapIntoReceivedMessage.php

```diff
@@ -29,7 +30,10 @@ public function receive(callable $handler): void
     {
         $this->decoratedReceiver->receive(function ($message) use ($handler) {
             if (null !== $message) {
-                $message = new ReceivedMessage($message);
+                if (!$message instanceof ConfiguredMessage) {
+                    $message = new ConfiguredMessage($message);
```

Comment:
Why didn't you use `ConfiguredMessage::from()` here?

Reply:
It was not deemed necessary to me for conveniency/readability here and will save this extra call this way.